pkg/component/watch.go

```diff
@@ -110,16 +119,15 @@ func addRecursiveWatch(watcher *fsnotify.Watcher, path string, ignores []string)
 		}
 		return nil
 	})
-	for _, v := range folders {
+	for _, folder := range folders {
 		ignore := false
-		for _, pattern := range ignores {
-			if matched, _ := regexp.MatchString(pattern, v); matched {
-				ignore = true
-				break
-			}
+
+		if matched, _ := isGlobExpMatch(directory, folder, ignores); matched {
+			ignore = true
```

Comment:
we don't need `ignore` variable anymore.
It should be enough to move `glog` and `continue` from below `if` statement here.
I don't see `ignore` used anywhere else